reminder_batch_scheduler/reminder_batch.py

```diff
@@ -0,0 +1,168 @@
+
+
+def build_action_rule_classifiers(wave, selected_batches):
+    action_rule_classifiers = {}
+    for action_type in constants.ACTION_TYPES_FOR_WAVE[wave]:
+        classifiers = constants.ACTION_TYPE_CLASSIFIERS[action_type]
+        classifiers['print_batch'] = selected_batches
+        action_rule_classifiers[action_type] = json.dumps(classifiers)
+    return action_rule_classifiers
+
+
+def select_batches(starting_batch, wave_classifiers, max_cases):
+    total_cases = 0
+    selected_batches = {}
+
+    for batch in range(starting_batch, 100):
+        batch_case_count = count_batch_cases(batch, wave_classifiers)
+        print(f'Batch: {batch}, Count: {batch_case_count}')
+        if total_cases + batch_case_count > max_cases:
+            break
+        total_cases += batch_case_count
+        selected_batches[str(batch)] = batch_case_count
+
+    return selected_batches
+
+
+def confirm_insert_rules():
+    confirmation = input(
+        colored('WARNING: You are about to write the the action rules to the database, '
+                'resulting in materials being sent for print. \nContinue? [Y/n] ', color='red'))
+    if confirmation != 'Y' and confirmation.lower() != 'yes':
+        return False
+    return True
+
+
+def generate_action_rules(action_rule_classifiers, action_plan_id):
+    action_rules = {}
+    for action_type, classifiers in action_rule_classifiers.items():
+        raw_trigger_date_time = input(f'Input trigger date time for action type {action_type} (RFC3339): ')
+        trigger_date_time = rfc3339.parse_datetime(raw_trigger_date_time)
+        action_rules[action_type] = (
+            "INSERT INTO actionv2.action_rule "
+            "(id, action_type, classifiers, trigger_date_time, action_plan_id, has_triggered) "
+            "VALUES (%s, %s, %s, %s, %s, %s);",
+            (str(uuid.uuid4()), action_type, classifiers, trigger_date_time, str(action_plan_id), False)
+        )
+    return action_rules
+
+
+def insert_action_rules(action_rules):
+    with db_helper.open_write_cursor() as db_cursor:
+        for action_type, action_rule in action_rules.items():
+            print("Inserting action rule for", action_type)
+            db_helper.execute_sql_query_with_write(db_cursor, action_rule[0], action_rule[1])
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(
+        description="Count cases for today's reminder batches to determine the batches we can schedule")
+    parser.add_argument('-w', '--wave',
+                        help='The reminder wave of contact',
+                        required=True,
+                        type=int,
+                        choices=constants.WAVE_CLASSIFIERS.keys())
+    parser.add_argument('-b', '--starting_batch',
+                        help='The batch number to start counting at',
+                        required=True,
+                        type=int,
+                        choices=range(1, 100))
```

Comment:
99 is the max print batch

Reply:
`range` is not inclusive on the end value